fastapi/exceptions.py

```diff
@@ -15,6 +15,9 @@ def __init__(
     ) -> None:
         super().__init__(status_code=status_code, detail=detail, headers=headers)
 
+    def __str__(self) -> str:
+        return f"HTTPException(status_code: {self.status_code}, detail: {self.detail})"
```

Comment:
Should this use `self.__class__.__name__` like in starlette (https://github.com/encode/starlette/blob/ada845cc8ad0b531b16749b9bf3a57114c60c679/starlette/exceptions.py#L22)

I haven't actually pulled this PR, but I think if I subclassed this class I'd end up with mismatching `__repr__` and `__str__`

```python
class CustomHTTPException(HTTPException):
    pass

exc = CustomHTTPException(status_code=400, detail="something went wrong")

print(exc)
#> HTTPException(status_code=400, detail='something went wrong')

print(repr(exc))
#> CustomHTTPException(status_code=400, detail='Something went wrong')
```